DogTales/src/components/physics.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <bave/app.hpp>
+#include <bave/graphics/sprite.hpp>
+
+namespace component {
+
+static constexpr glm::vec2 max_velocity{1000.f, 1000.f};
+static constexpr float tick_limit{0.03f}; //this should possibly go somewhere else and be globally accessible
```

Comment:
This represents time and should use chrono units (`bave::Seconds`).

Reply:
I also think 30ms is too little, 100ms is ok as the death-spiral-bail-out value.
```suggestion
constexpr glm::vec2 max_velocity_v{1000.f};
constexpr bave::Seconds tick_limit_v{100ms};
```

Reply:
fixed!